app/templates/books/book/pages/page/edit.hbs

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
You said it is remove in this PR (in another PR :)) ✂️

Reply:
Im certain that i removed it, so probably it was removed in the limbo commits xd